docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListOrganizationMembers
+
+List the members and invites of the organization that you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(Tuple[List[viam.proto.app.OrganizationMember]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.OrganizationMember): A tuple containing to lists:
+  - The first (`[0]`) is a list of organization members.
+  - The second (`[1]`) is a list of organization invites.
+
+```python {class="line-numbers linkable-line-numbers"}
+member_list = await cloud.list_organization_members()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organization_members).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetOrganizationNamespaceAvailability
+
+Check the availability of an organization namespace.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `public_namespace` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The organization namespace to check.
+Namespaces can only contain lowercase alphanumeric and dash characters.
+
+**Raises:**
+
+- GRPCError: An error is raised if an invalid namespace (for example, `""`) is provided.
+
+**Returns:**
+
+- [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): `True` if the provided namespace is available.
+
+```python {class="line-numbers linkable-line-numbers"}
+available = await cloud.get_organization_namespace_availability(public_namespace="my-cool-organization")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_organization_namespace_availability).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### UpdateOrganizationInviteAuthorizations
+
+Update the authorizations attached to an organization invite that has already been created.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `email` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Email of the user the invite was sent to.
+- `add_authorization` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Optional list of authorizations to add to the invite.
+- `remove_authorization` [(Optional[List[viam.proto.app.Authorization]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Authorization): Optional list of authorizations to remove from the invite.
+
+**Raises:**
+
+- GRPCError: An error is raised if no authorizations are passed or if an invalid combination of authorizations is passed (for example, an authorization to remove when the invite only contains one authorization).
```

Comment:
Again would move the example out here

Reply:
Same thing as above-- if error is being raised/returned just describe the error in the directive tense for the user instead of describing it like this

Reply:
```suggestion
- `GRPCError`: This error is raised if no authorizations are passed or if an invalid combination of authorizations is passed (for example, an authorization to remove when the invite only contains one authorization).
```
Is this better? Also I think the example could be helpful here?

Reply:
I don't mean not having the example, just moving it out so that it's in the method description (line 140). Would suggest that so people can more easily see that functionality of this method

Reply:
Hmm I think it's good to keep it here but can add it to the description as well--what do you think of my new suggestion on 140?

Reply:
LGTM!